gulpfile.js

```diff
@@ -1,56 +1,78 @@
 var gulp = require('gulp'),
-    bump = require('gulp-bump'),
-    git = require('gulp-git'),
-    inject = require('gulp-inject-string'),
-    merge = require('merge-stream'),
-    tag_version = require('gulp-tag-version'),
-    tslint = require('gulp-tslint'),
-    typings = require('gulp-typings'),
-    shell = require('gulp-shell');
+  bump = require('gulp-bump'),
+  git = require('gulp-git'),
+  inject = require('gulp-inject-string'),
+  merge = require('merge-stream'),
+  tag_version = require('gulp-tag-version'),
+  tslint = require('gulp-tslint'),
+  typings = require('gulp-typings'),
+  shell = require('gulp-shell');
 
 var paths = {
-  src_ts: "src/**/*.ts",
-  tests_ts: "test/**/*.ts"
+  src_ts: 'src/**/*.ts',
+  tests_ts: 'test/**/*.ts',
 };
 
 function versionBump(semver) {
-  return gulp.src(['./package.json'])
-    .pipe(bump({type: semver}))
+  return gulp
+    .src(['./package.json'])
+    .pipe(bump({ type: semver }))
     .pipe(gulp.dest('./'))
     .pipe(git.commit('bump package version'))
     .pipe(tag_version());
 }
 
-gulp.task('typings', function () {
-  return gulp.src('./typings.json')
-    .pipe(typings());
+gulp.task('typings', function() {
+  return gulp.src('./typings.json').pipe(typings());
 });
 
 gulp.task('typings-vscode-definitions', ['typings'], function() {
   // add vscode definitions
   return gulp.src('./typings/index.d.ts').pipe(gulp.dest('./typings'));
-})
+});
 
 gulp.task('tslint', function() {
   var tslintOptions = {
-    summarizeFailureOutput: true
+    summarizeFailureOutput: true,
   };
 
-  var srcs = gulp.src(paths.src_ts)
+  var srcs = gulp
+    .src(paths.src_ts)
     .pipe(tslint({ formatter: 'verbose' }))
     .pipe(tslint.report(tslintOptions));
-  var tests = gulp.src(paths.tests_ts)
+  var tests = gulp
+    .src(paths.tests_ts)
     .pipe(tslint({ formatter: 'verbose' }))
     .pipe(tslint.report(tslintOptions));
   return merge(srcs, tests);
 });
 
-gulp.task('default', ['tslint', 'compile']);
+gulp.task(

```

Comment:
Mainly because that package isn't maintained, and it's missing the "es5" trailing commas option.